### ISSUE TITLE
UI: Default type in sitemap adds extra frame

### ISSUE THREAD

**Issue by metbril** [eclipse/smarthome#2933]
An item of type Player added in a sitemap as Default adds an extra frame to the sitemap when it is the first entry in a frame. This applies to both basic ui and classic ui

items
```
Player Sonos_Living_Control <player>
Dimmer Sonos_Living_Volume <soundvolume>
```
partial sitemap
```
Text label="Demo" {
  Frame label="First (ok)" {
    Slider  item=Sonos_Living_Volume
    Default item=Sonos_Living_Control
  }
  Frame label="Second (extra frame)" {
    Default item=Sonos_Living_Control
    Slider  item=Sonos_Living_Volume
  }
}
```
![extraframe](https://cloud.githubusercontent.com/assets/559346/22441644/917bc476-e738-11e6-8cfd-728eaeaf4ba4.png)

**Comment by lolodomo** [comment]
The bug is here: https://github.com/eclipse/smarthome/blob/master/extensions/ui/org.eclipse.smarthome.ui.basic/src/main/java/org/eclipse/smarthome/ui/basic/internal/render/PageRenderer.java#L107
A default element in sitemap leads to a widget without parent. The test is in this case not appropriate.

**Comment by lolodomo** [comment]
@sjka @kaikreuzer  : please close this issue, the merged PR #3471 fixed it.
